pkg/host/internal/sriov/sriov.go

```diff
@@ -1011,13 +1019,46 @@ func (s *sriov) setEswitchModeAndNumVFs(pciAddr string, desiredEswitchMode strin
 			return err
 		}
 	}
+
 	if err := s.SetSriovNumVfs(pciAddr, numVFs); err != nil {
 		return err
 	}
 
 	if desiredEswitchMode == sriovnetworkv1.ESwithModeSwitchDev {
 		return s.setEswitchMode(pciAddr, sriovnetworkv1.ESwithModeSwitchDev)
 	}
+
+	return nil
+}
+
+func (s *sriov) setEswitchModeAndNumVFsIce(pciAddr string, desiredEswitchMode string, numVFs int) error {
```

Comment:
please add a function comment to explain the process of how we configure switchDev virtual functions for this case